### PR TITLE
Feature/display loggedin user

### PR DESCRIPTION
**Pull request by Tachi-67**
This feature is built on top of feature/log_out, because I need to use the overviewViewModel.

With this feature the user can see on the overview screen the email used to log in.

We could probably in the future change it to the real name if necessary.

This took me ~1 hour to finish.

### CODE REVIEW

File: app/src/main/java/ch/epfl/sdp/cook4me/ui/overview/OverviewScreen.kt [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+            }
+        }
+    }
+}
+
+@Composable
+fun CurrentLoggedInEmailText(email: String?) {
+    var userEmail = email
+    if (userEmail == null) {
+        userEmail = "Empty Email"
```
Comment (dayan9265) [inline app/src/main/java/ch/epfl/sdp/cook4me/ui/overview/OverviewScreen.kt:105]: I think this should be in the string resource

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -8,6 +8,7 @@
     <string name="navigate_to_add_event">Add event</string>
     <string name="navigate_to_add_signup">Sign up</string>
     <string name="navigate_to_postView">Post</string>
+    <string name="sign_out">Sign out</string>
```
Comment (dayan9265) [inline app/src/main/res/values/strings.xml:11]: It appears to be unused this string, I don't know if you want to later use it or it remains from a previous implementation?

**Review by dayan9265 — CHANGES_REQUESTED** [review]
This code looks good to me 👍, just requested small change on the string resource, but otherwise well done :)

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (Tachi-67) [reply app/src/main/res/values/strings.xml:11]: It is a name from another branch, I merged it into this branch accidentally ^^', it's just fine, they will finally be used after I merged everything.

File: app/src/main/java/ch/epfl/sdp/cook4me/ui/overview/OverviewScreen.kt (hunk 1 above)
Comment (Tachi-67) [reply app/src/main/java/ch/epfl/sdp/cook4me/ui/overview/OverviewScreen.kt:105]: Good idea! Will do.

**Review by dayan9265 — APPROVED** [review]
Nice, thx for the small modif, you should be able to merge it now :)

**Pull request merged**